test/test_ops.py

```diff
@@ -712,7 +713,7 @@ def check_tensor_floating_is_differentiable(t):
 
     # Reference testing for operations in complex32 against complex64.
     # NOTE: We test against complex64 as NumPy doesn't have a complex32 equivalent dtype.
-    @ops(op_db, allowed_dtypes=(torch.complex32,))
+    @ops(op_db, dtypes=OpDTypes.supported, allowed_dtypes=(torch.complex32,))
```

Comment:
UnaryUfuncInfo specify a limited default_test_dtypes which doesn't include `complex32` (if default_test_dtypes is None then supported types are used)

https://github.com/pytorch/pytorch/blob/be354d8139f70c5fabfce882671ad1aed057b1ba/torch/testing/_internal/common_methods_invocations.py#L1349-L1351

Reply:
Just add complexhalf to the default_test_dtypes

Reply:
I think that could be problematic later if a new class is added which specifies `default_test_dtypes` without `complex32`. This reference test would be skipped silently even if the op claims to support `complex32`.

Reply:
OK, new proposal: let's just remove `default_test_dtypes`. The test suite hasn't continued to evolve in that direction. It only affects elementwise unary ops, after all.

Reply:
No sugars!

Reply:
Spectral Ops (ffts) also specify this field. Will file an issue to remove this field from OpInfo in the future.

Reply:
`default_test_dtypes` are removed now